modules/client/src/main/scala/ch/epfl/bluebrain/nexus/iam/client/types/AuthToken.scala

```diff
@@ -0,0 +1,15 @@
+package ch.epfl.bluebrain.nexus.iam.client.types
+
+import akka.http.scaladsl.model.headers.OAuth2BearerToken
+
+/**
+  * A data structure which represents a token
+  *
+  * @param value the token value
+  */
+final case class AuthToken(value: String)
```

Comment:
I don't see the point of this abstraction if we then use implicit conversions everywhere.

Reply:
To hide `akka` stuff

Reply:
I'm not fond of using the akka-http data types in the client api; if something needs to go, I would prefer to get rid of the implicit conversions.